.github/workflows/local-interchain.yaml

```diff
@@ -0,0 +1,30 @@
+name: Local Interchain
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "main"      
```

Comment:
If we are running on PR's, I don't think this is necessary.

Reply:
Resolved in dafd9c610d57cfb9fe8f144b1d67cc41690141aa